raiden/raiden_service.py

```diff
@@ -59,6 +59,27 @@
 log = structlog.get_logger(__name__)  # pylint: disable=invalid-name
 
 
+def _redact_secret(data):
```

Comment:
Same discussion as before: Didn't we say we don't want to do this as these are possibly  big objects?

Reply:
This will probably be used with small objects (state changes and events).